tests/modules/datatype2.c

```diff
@@ -415,7 +415,7 @@ int MemUsage_RedisCommand(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int a
     }
 
     if (dbid < 0 || dbid >= MAX_DB) {
-        return ValkeyModule_ReplyWithError(ctx, "ERR dbid out of range");
+        return ValkeyModule_ReplyWithError(ctx, "ERR dbid out-of-range");
```

Comment:
This is a user error message, so we should avoid changing it in case anyone has taken a character-by-character dependency on it.